openedx/features/effort_estimation/block_transformers.py

```diff
@@ -0,0 +1,225 @@
+"""
+Effort Estimation Transformer implementation.
+Adds effort estimations for block types it recognizes.
+"""
+
+import math
+
+import crum
+import lxml.html
+from django.utils.functional import cached_property
+from edxval.api import get_videos_for_course
+
+from openedx.core.djangoapps.content.block_structure.transformer import BlockStructureTransformer
+from openedx.core.lib.mobile_utils import is_request_from_mobile_app
+
+from .toggles import EFFORT_ESTIMATION_LOCATION_FLAG
+
+
+class EffortEstimationTransformer(BlockStructureTransformer):
+    """
+    A transformer that adds a partition condition for all graded content
```

Comment:
is this comment accurate? 
Also could we include in the docstring a description of what the feature is and perhaps a link to a product doc on it and there is one that is publicly available?

Reply:
Ah yeah just a copy / paste error. Will update.